loggly/loggly.node

```diff
@@ -0,0 +1,182 @@
+{
+  "_exportDate": "2018-08-10T21:49:40.619Z",
+  "_type": "flow",
+  "applicationId": "5b6b6ac661b32f0007645f58",
+  "customNodeConfig": {
+    "fields": [
+      {
+        "description": "[Customer Tokens](http://loggly.com/docs/customer-token-authentication-token/) are used by Loggly to associate data you send with your account. The token is used as part of the API request.",
+        "id": "token",
+        "label": "Loggly Customer Token",
+        "required": true,
+        "type": "stringTemplate"
+      },
+      {
+        "description": "The payload path of event data to send to Loggly.",
+        "id": "payloadPath",
+        "label": "Event Data",
+        "required": true,
+        "type": "payloadPath"
+      }
+    ],
+    "outputCount": 1,
+    "resultDescription": "The result of the [Loggly HTTP API](https://www.loggly.com/docs/http-endpoint/) request. \n\nIf the request fails, the result will be an object with an \"error\" property.",
```

Comment:
These newlines will be stripped because it's markdown. Can probably just remove them.